.github/workflows/publish-deb-package.yml

```diff
@@ -80,15 +80,16 @@ jobs:
           HZ_LICENSEKEY: ${{ secrets.HZ_LICENSEKEY }}
         run: |
           source ./common.sh
-          wget \
-            https://repository.hazelcast.com/api/gpg/key/public \
-            --header "Authorization: Bearer ${{ env.JFROG_TOKEN }}" \
-            --output-document - \
-            --quiet \
-            | gpg --dearmor \
-            | sudo tee /usr/share/keyrings/hazelcast-archive-keyring.gpg > /dev/null
-          echo "deb [signed-by=/usr/share/keyrings/hazelcast-archive-keyring.gpg] ${DEBIAN_REPO_BASE_URL} ${PACKAGE_REPO} main" | sudo tee -a /etc/apt/sources.list
-          sudo apt-get update && sudo apt-get install -y ${{ env.HZ_DISTRIBUTION }}=${HZ_VERSION}
+          apt-get install -y curl gettext-base gpg sudo wget \
```

Comment:
In the docs we have different command :( 
`apt-get install -y --no-upgrade wget gpg coreutils`

Reply:
Me right now: https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExYmp3aDNvZzNuYTZlb2ZjeXU0dnRpY3U1bHpoczhmeTQwcDY0MmlkaSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/PUS7ynUCR6itJgWppA/giphy.gif